packages/amqp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@message-queue-toolkit/amqp",
-  "version": "12.0.3",
+  "version": "13.0.0",
```

Comment:
I think all are major release due to API name change, but please let me know if I am wrong